extensions/amp-next-page/amp-next-page.md

```diff
@@ -51,10 +51,14 @@ limitations under the License.
 
 ## Behavior
 
-Given a list of pages, `amp-next-page` tries to load them after the current document, providing an infinite-scroll type experience. 
+Given a list of pages, `amp-next-page` tries to load them after the current 
+document, providing an infinite-scroll type experience. 
 
-The pages should be inlined using a JSON format.
+Pages should be specified in a JSON config loaded from a remote URL listed in
+the element `src`, or inlined as a `script` element child.
 ```html
+<amp-next-page src="https://example.com/next-page-config.json"></amp-next-page>
```

Comment:
Is this supported as of now?  Awesome, if so!

Reply:
Not merged yet: https://github.com/ampproject/amphtml/pull/16624

I'll split this into a separate commit and send another PR after that hits prod.